src/components/Onboarding/content.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import large1 from './highlights/assets/rinkeby/large/1.png'
+import large2 from './highlights/assets/rinkeby/large/2.png'
+import large3 from './highlights/assets/rinkeby/large/3.png'
+import large4 from './highlights/assets/rinkeby/large/4.png'
+import large5 from './highlights/assets/rinkeby/large/5.png'
+
+import small1 from './highlights/assets/rinkeby/small/1.png'
+// import small2 from './highlights/assets/rinkeby/small/2.png'
+import small3 from './highlights/assets/rinkeby/small/3.png'
+import small4 from './highlights/assets/rinkeby/small/4.png'
+import small5 from './highlights/assets/rinkeby/small/5.png'
+
+const highlights = {}
+highlights.latest = [
+  {
+    title: {
+      small: null,
+      large: 'Welcome, juror',
+    },
+    description: {
+      small: null,
+      large: `This is a preview release of the official Court Dashboard that will launch on Feb 10th. We'd love for you to explore it and tell us what you think!`,
+    },
+    visual: {
+      small: small1,
+      large: large1,
+    },
+  },
+  {
+    title: {
+      small: null,
+      large: 'Getting Rinkeby ANJ',
+    },
+    description: {
+      small: null,
+      large: (
+        <span>
+          We've airdropped 10,000 test ANJ on Rinkeby to the same Ethereum
+          account you registered with on mainnet. You can use this test ANJ to
+          become an active juror and participate in the arbitration process.
+          <br />
+          <br />
+          <br />
+          <b>Include link to faucet once we have it</b>
```

Comment:
Reminder to update this :).

Reply:
I removed it for now. Do we need a faucet at all? The content says the ANJ has been airdropped already :thinking:

Reply:
The intention was for people who hadn't registered on mainnet to use the faucet but we can just do it manually for now :).